cms/static/cms/js/modules/cms.base.js

```diff
@@ -118,8 +120,8 @@ var CMS = {
             },
 
             // handles the tooltip for the plugins
-            showTooltip: function (name, id) {
-                var tooltip = $('.cms-tooltip');
+            showTooltip: function (e, name, id) {
+                var tooltip = CMS.API.Helpers.pickTooltip();
```

Comment:
yeah we also plan to do that for messages, I thing those components should be handled separately